CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## v1.6.12
```

Comment:
let's keep the version as 1.6.11, it's basically the same work, e.g. minutes later